mypyc/test-data/run-misc.test

```diff
@@ -13,17 +13,16 @@ async def g() -> int:
 async def f() -> int:
     return await g()
 
-loop = asyncio.get_event_loop()
-result = loop.run_until_complete(f())
+result = asyncio.run(f())
 assert result == 1
 
 [typing fixtures/typing-full.pyi]
 
 [file driver.py]
 from native import f
 import asyncio
-loop = asyncio.get_event_loop()
-result = loop.run_until_complete(f())
+
+result = asyncio.run(f())
```

Comment:
Unfortunately `asyncio.run` doesn't exist yet in 3.6 which we still support. You may have to put an `if sys.version_info >= (3, 7)` check here.